compiler/optimizer/LoopReducer.cpp

```diff
@@ -1088,23 +1088,24 @@ TR_LoopReducer::generateArraycopy(TR_InductionVariable * indVar, TR::Block * loo
    if (arraycopyLoop.getStoreAddress()->getIncrement() < 0)
       //&& (storeAddr->getFirstChild()->getSymbol()->getRegisterMappedSymbol() == loadAddr->getFirstChild()->getSymbol()->getRegisterMappedSymbol()))
       {
-      if (comp()->cg()->getSupportsPrimitiveArrayCopy())
-         arraycopy->setBackwardArrayCopy(true); /* bit available only to primitive arraycopy */
```

Comment:
This comment should be preserved somewhere because it is a warning not to copy the flag to reference copying code.